torchgeo/datasets/globbiomass.py

```diff
@@ -0,0 +1,282 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""GlobBiomass dataset."""
+
+import glob
+import os
+from typing import Any, Callable, Dict, Optional
+
+import matplotlib.pyplot as plt
+from rasterio.crs import CRS
+from torch import Tensor
+
+from .geo import RasterDataset
+from .utils import BoundingBox, check_integrity, extract_archive
+
+
+class GlobBiomass(RasterDataset):
+    """GlobBiomass dataset.
+
+    The `GlobBiomass dataset <https://doi.pangaea.de/10.1594/PANGAEA.894711>`_
+    consists of global pixel wise aboveground biomass (AGB) and growth stock
+    volume (GSV) maps.
+
+    Dataset features:
+
+    * estimates of AGB and GSV around the world at ~100m per pixel resolution
+      (45,000x45,0000 px)
+    * standard error maps of respective measurement at same resolution
+
+    Dataset format:
+
+    * estimate maps are single-channel
+    * standard error maps are single-channel
+
+    The data can be manually downloaded from `this website
+    <https://globbiomass.org/wp-content/uploads/GB_Maps/
+    Globbiomass_global_dataset.html>`_.
+
+    If you use this dataset please cite it with the following citation:
+
+    * Santoro, M. et al. (2018): GlobBiomass - global datasets of forest biomass.
+      PANGAEA, https://doi.org/10.1594/PANGAEA.894711
+
+    .. versionadded:: 0.3
+    """
+
+    is_image = False
```

Comment:
Should probably add:
```python
filename_glob = "*_agb*"
```
to avoid picking up any other datasets in the same directory. Judging from the test data that string is in all files? If not, we should prob add gsv files to the fake test data.